AddonCatalog.json

```diff
@@ -262,12 +270,14 @@
       "zip_url": "https://github.com/triplus/CommandPanel/archive/refs/heads/master.zip"
     }
   ],
-    "Color-Palette-Theme" : [{
-        "branch_display_name" : "main" ,
-        "repository" : "https://github.com/altangarts/FreeCAD-Themes-ColorPalette" ,
-        "zip_url" : "https://github.com/altangarts/FreeCAD-Themes-ColorPalette/archive/refs/heads/main.zip" ,
-        "git_ref" : "main"
-    }],
+  "Color-Palette-Theme": [
+    {
+      "branch_display_name": "main",
+      "repository": "https://github.com/altangarts/FreeCAD-Themes-ColorPalette",
+      "zip_url": "https://github.com/altangarts/FreeCAD-Themes-ColorPalette/archive/refs/heads/main.zip",
+      "git_ref": "main"
+    }
+  ],
```

Comment:
Just wondering why this portion was modified ?

Reply:
I opened the repository in vscode and have it configured to format files on save. It must have reformatted this file this I changed it and only this entry was modified. The braces were right next to the square brackets before. I can't undo this change, if you want